src/commands/spawn.ts

```diff
@@ -20,6 +20,7 @@ export default class Spawn extends Command {
     grace: Flags.integer({ char: 'g', description: 'optional grace period (in seconds) to wait after last connection is closed before shutting down container' }),
     port: Flags.integer({ char: 'p', description: 'optional port for jamsocket to proxy requests to (default is 8080)', hidden: true }),
     tag: Flags.string({ char: 't', description: 'optional tag for the service to spawn (default is latest)' }),
+    requireBearerToken: Flags.boolean({ char: 'r', description: 'require a bearer token to access the service' }),
```

Comment:
Does this mean the CLI flag is `--requireBearerToken` or `--require-bearer-token`?

Also, maybe the description is something like "generate a bearer token for accessing the backend"?

Reply:
Ah, I didn't realize oclif exposed the name. I've changed it to `--require-bearer-token` and added to the description.